docs/lib/content/commands/npm-access.md

```diff
@@ -16,29 +16,10 @@ For all of the subcommands, `npm access` will perform actions on the packages
 in the current working directory if no package name is passed to the
 subcommand.
 
-* public / restricted (deprecated):
-  Set a package to be either publicly accessible or restricted.
-
 * grant / revoke (deprecated):
   Add or remove the ability of users and teams to have read-only or read-write
   access to a package.
```

Comment:
Note that this claims _something_ is deprecated, however both `npm access grant` and `npm access revoke` appear to be alive and well in npm 10.8.2 (unlike the other items which I'm removing).

Reply:
Ok. 655752f